paper-slider.html

```diff
@@ -292,7 +292,7 @@
      * @method increment
      */
     increment: function() {
-      this.value = this._clampValue(this.value + this.step);
+      this.value = this._clampValue(this.value + this.step).toString();
```

Comment:
I think one possible fix may require to rename `value` in `IronRangeBehavior` to something like `numericValue`